### PR TITLE
(DOCSP-15923): Call out "delta sync"

### PR DESCRIPTION
**Pull request by cbush**
### Jira

https://jira.mongodb.org/browse/DOCSP-15923

### Staged Changes (Requires MongoDB Corp SSO)

https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/delta/sync/protocol/#changeset

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sync/protocol.txt [hunk 1]
```diff
@@ -49,6 +49,14 @@ client (including changes in a synced MongoDB cluster) at any time and uses an
 operational transformation algorithm to serialize changes into a linear order
 and resolve conflicting changesets before sending them to connected clients.
 
+.. _delta-sync:
+
+.. note:: Delta Sync
+
+   To maximize bandwidth in mobile networks, the {+sync+} service only
+   transmits the changes (deltas) at the field level. The service
+   compresses the deltas on the fly with zlib compression.
```
Comment (mongomoe) [inline source/sync/protocol.txt:58]:
```suggestion
   compresses the deltas on the fly with :wikipedia:`zlib <Zlib>` compression.
```

File: source/sync/protocol.txt [hunk 2]
```diff
@@ -49,6 +49,14 @@ client (including changes in a synced MongoDB cluster) at any time and uses an
 operational transformation algorithm to serialize changes into a linear order
 and resolve conflicting changesets before sending them to connected clients.
 
+.. _delta-sync:
+
+.. note:: Delta Sync
+
+   To maximize bandwidth in mobile networks, the {+sync+} service only
```
Comment (mongomoe) [inline source/sync/protocol.txt:56]: It took me a few re-reads to understand what the first sentence was saying. I think it would be a bit simpler to understand if we avoided overloading the paragraph with technical phrases like "maximize bandwidth" and "deltas". I think we need to have the phrase "delta sync" at least once for SEO/search? Here's an alternative that attempts to accomplish that in simpler terms that still, hopefully, satisfies the keyword/search requirement:

"Realm Sync uses a process known as ``delta sync`` that transmits only the changes to the altered fields in order to synchronize data faster. The service compresses the changes on the fly with zlib compression."

**Review by mongomoe — APPROVED** [review]
LGTM but 1 link suggestion and 1 wording suggestion. Technically this PR satisfies the ticket but given that it took me a couple re-reads to understand, I can see others needing to re-read this as well.

File: source/sync/protocol.txt (hunk 2 above)
Comment (cbush) [reply source/sync/protocol.txt:56]: Thanks for the feedback. I have completely reworked it. Let me know what you think!

Comment (mongomoe) [reply source/sync/protocol.txt]: New wording is much clearer, double LGTM @cbush
